stack/README.md

```diff
@@ -7,6 +7,27 @@ For information on how to configure the stack, see the [tobs documentation](http
 ## Applying custom configuration
 
 To allow customization of the stack and allow easy `git rebase` flow it is recommended
-to use `values-overrides.yaml` file. This file is ignored by git and can be used to override any configuration option in the stack while allowing to keep the original configuration in the `values.yaml` file. It also allows easy sharing of the configuration between different users.
+to use `values-overrides.yaml` file. This file is by ignored by git and can be used to override any configuration option in the stack while allowing to keep the original configuration in the `values.yaml` file. It also allows easy sharing of the configuration between different users.
+
+The file doesn't exist by default and will be created on first run of `helm install` command. Below is an example of the file:
+
+```yaml
```

Comment:
Shall we keep this as template file and create values-overrides.yaml if it didn't exists?

```makefile
values-override.yaml: values-override.yaml.in
   @diff -urN values-override.yaml values-override.yaml.in  | patch -p1
```

The above one creates values-override.yaml didn't exist, otherwise it merges with upstream if there is a change due to pull.

WDYT?

Reply:
Per design, this file shouldn't have any upstream changes. Any upstream changes should be placed in `values.yaml`

Reply:
Right, how about using only values-overrides.yaml and keeping values.yaml as template? Less merging hassels right?

Reply:
That's the idea :) When testing stuff user is supposed to change parameters. At the end of benchmarks we should have  some results and change benchmark suite accordingly to allow easier usage for others. With that approach `values-overrides.yaml` are useful for changing things during benchmarks and sharing untested things as we go and `values.yaml` give us long-term the baseline which should be amended after benchmarks to improve future benchmarks.